### DISCUSSION TITLE
Cluster reconfiguration

### DISCUSSION THREAD

**Discussion by Flybro** [cloudnative-pg/cloudnative-pg#206] [Q&A] [answered]
I have a question, after changes in the configuration of an already operating cluster, do I have to do something else so that the changes are noticed or the operator will catch them himself? In this particular case, I turned on pod's monitoring (enablePodMonitor).

Cheers,
BB

**Comment by sxd** [comment, marked as answer]
Hi @Flybro 

You don't need to do anything, if the PodMonitor resource is available in the cluster (you have the Prometheus operator installed) CloudNativePG will detect it and then create the resource for you, if the PodMonitor resource is available after it was enabled in the cluster you will have to wait for the next reconciliation loop to create everything.

Hope this answer your question, please let us know if you need more help!

CheerS!
